composer.lock

```diff
@@ -9,30 +9,30 @@
     "packages-dev": [
         {
             "name": "doctrine/instantiator",
-            "version": "1.5.0",
+            "version": "2.0.0",
```

Comment:
Version 2.0.0 requires php 8, so leaving this on 1.5.0 should fix the workflow error.